jakartaee9/src/web-jsptaglibrary_3_0.xsds

```diff
@@ -0,0 +1,1124 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2003, 2020 Oracle and/or its affiliates. All rights reserved.
+
+    This program and the accompanying materials are made available under the
+    terms of the Eclipse Public License v. 2.0, which is available at
+    http://www.eclipse.org/legal/epl-2.0.
+
+    This Source Code may also be made available under the following Secondary
+    Licenses when the conditions for such availability set forth in the
+    Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+    version 2 with the GNU Classpath Exception, which is available at
+    https://www.gnu.org/software/classpath/license.html.
+
+    SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+
+-->
+
+<xsd:schema
+     targetNamespace="http://xmlns.eclipse.org/xml/ns/jakartaee"
```

Comment:
Please change http://xmlns.eclipse.org/xml/ns/jakartaee to https://jakarta.ee/xml/ns/jakartaee

Reply:
Done